bootstrap-joining-demo/marathon-api/marathon/app.json

```diff
@@ -17,9 +21,36 @@
       "name": "akkamgmthttp"
     }
   ],
+  "readinessChecks": [{
+    "path": "/ready",
+    "portName": "http",
+    "protocol": "HTTP",
+    "gracePeriodSeconds": 300,
+    "intervalSeconds": 10,
+    "timeoutSeconds": 5,
+    "maxConsecutiveFailures": 3,
+    "ignoreHttp1xx": false
+  }],
+  "healthChecks": [
+    {
+      "path": "/healthy",
+      "portName": "http",
+      "protocol": "HTTP",
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 10,
+      "timeoutSeconds": 5,
+      "maxConsecutiveFailures": 3,
+      "ignoreHttp1xx": false
+    }
+  ],
+  "upgradeStrategy": {
+    "maximumOverCapacity": 0,
+    "minimumHealthCapacity": 0.5
+  },
+  "killSelection": "YOUNGEST_FIRST",
```

Comment:
oh this is very nice! Glad they have such strategy, matches what we need totally